resources/decision_tree.json

```diff
@@ -12,12 +12,103 @@
         ]
       },
```

Comment:
Thoughts:
- Split tree based on variant type: structural variant vs sequence variant
- Split sequence variants based on effect
  - effect 
    - missense
    - splice (splice_acceptor_variant/splice_donor_variant/splice_region_variant/splice_donor_5th_base_variant/splice_donor_region_variant/splice_polypyrimidine_tract_variant)
    - other
  - Use SIFT and PolyPhen nodes only in missense branch
  - Use SpliceAI only in splice branch
- Combine SIFT and PolyPhen in one MULTI_BOOL node to reduce VUS/LP/P misses
- Never classify VKGL LP/P variants below their VKGL class
- Be conservative in classifying ClinVar LP/P variants below their ClinVar class